### ISSUE TITLE
[a11y] Use aria-label alongside titles

### ISSUE THREAD

**Issue by maxxxxxdlp** [specify/specify7#960]
Fixed in https://github.com/specify/specify7/commit/f1988806772e09cacf87fa218f6fe8313656b07f and https://github.com/specify/specify7/commit/65a25b3f7da6706becdcc6eb940daf68f5fcb2f9 and https://github.com/specify/specify7/commit/10f084e1b907b5d155a542d51396866c3b765725

**Comment by grantfitzsimmons** [comment]
How do I test this?

**Comment by maxxxxxdlp** [comment]
This would be part of the global accessibility testing.
This issue in particular makes sure that if an input field does not have a visible label, (i.e, the search box in the header), the label is still accessible to the screen reader
